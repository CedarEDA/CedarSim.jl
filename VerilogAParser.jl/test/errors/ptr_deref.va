module ptr_deref(p, n);
    analog begin
        p=MTRLMOD->x;
    end
endmodule
