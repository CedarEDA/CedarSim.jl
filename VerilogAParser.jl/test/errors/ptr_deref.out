[31m[1mâ•­ ERROR [VAS99]:[22m[39m[31m Unexpected operator.[39m
[31mâ”‚[39m[90m  â•­â”€â”€â”€â”€â”€ ptr_deref.va:3[39m
[31mâ”‚[90m3[0m [90mâ”‚[0m         p=MTRLMOD->x;
[31mâ”‚[0m  [90mâ”Š[0m                   [91mâ”“
[31mâ”‚[0m  [90mâ”Š[0m        [91mâœ— Unexpected operator: [0m
[31mâ”‚[39m[90m  â”‚[39m
[31mâ”‚[39m[90m  â•°â”€â”€â”€â”€â”€ [39m
[31mâ•¯[39m